src/Uno.UI/UI/Xaml/Application.wasm.cs

```diff
@@ -68,6 +68,8 @@ private void Initialize()
 				OnLaunched(new LaunchActivatedEventArgs(ActivationKind.Launch, arguments));
 			}
 		}
+
+		private ApplicationTheme GetDefaultSystemTheme() => ApplicationTheme.Light;
```

Comment:
Maybe we can use that?
<https://drafts.csswg.org/mediaqueries-5/#prefers-color-scheme>

Reply:
Good find! I will look into it

Reply:
https://caniuse.com/#feat=prefers-color-scheme

Reply:
Done :-) !!!

Reply:
I had to add a boolean flage _initializationCompleted to check if the setter should throw, as behavior in UWP allows to set the ApplicationTheme any time during the constructor (even at the end), so the first place it should throw is during `OnLaunched`